### ISSUE TITLE
Ventoy won't boot on a Surface Pro 6

### ISSUE THREAD

**Issue by ilkevinli** [ventoy/Ventoy#127]
Ventoy will not boot on a Surface Pro 6 with secure boot disabled. I know the drive is good because my desktop boots fine with it. I also tried with secure boot on, and that also fails to boot.

**Comment by SamLukeYes** [comment]
See my issues #119 #87

**Comment by harshpatel0** [comment]
Hey, I also have the same issue on my Surface Pro 6, it will boot when Secure Boot is enabled but the Surface Type Cover port does not work and the built in keyboard in the UEFI does not work, it seems like the system just plain locks up as no presses on the on screen keyboard work, including closing it, pressing ENTER and so on, I tried disabling secure boot but then it refuses to boot and gets stuck on the Surface Logo, I have tried this with 2 different flash drives, both work on my other laptop (with secure boot turned off) just fine, I have tried disabling Alternate Boot Option as well and that didn't work either.
Using bootable USB drives burnt through Rufus does work.

**Comment by Frontesque** [comment]
My surface pro6 experiences 2 issues depending on whether secure boot is enabled or disabled.

Secure boot disabled:
Shows the windows screen and doesn't get any further than this. It's like it's frozen. The only way to get out of this state is to hold the power button for 10 seconds. It doesn't matter if I let it sit for 10 minutes or 2 hours. It is stuck here.
![PXL_20241123_180730844](https://github.com/user-attachments/assets/ca14afeb-aeb2-43a0-9ec5-016eac21ddd1)

Secure boot enabled (ms & 3rd party ca)
It boots past the windows screen, however freezes and becomes unresponsive at the security violation. Same issue, which is completely unresponsive. No key presses work, nothing. Must hold power button for 10 seconds.
![PXL_20241123_180851014](https://github.com/user-attachments/assets/dad8b3ea-ef7b-408b-8b24-0b279c737e0f)



It seems like ventoy is completely broken on the sp6 currently.

**Comment by harshpatel0** [comment]
Yup, @Frontesque that's the same issues I'm having, thank you for describing it way better than I could with pictures too.
It's almost like the system just locks up after a few seconds, a previous version of Ventoy did work tho, I don't remember which one it was unfortunately.
